vue-app/src/components/UI/Recursive.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="recursive-component">
+        <p>{{ props.message }} {{ props.depth }}</p>
+        <Recursive
+        v-if="props.depth < props.maxDepth"
+        :message = "props.message"
+        :depth = "props.depth + 1"
+        :maxDepth = "props.maxDepth"
+        />
+    </div>
+</template>
+
+<script setup lang="ts">
+import type { RecursiveProps } from "../../models/RecursiveProps"
+import { defineProps, defineComponent } from "vue";
+import Recursive from "./Recursive.vue"
```

Comment:
this component name is against the convention

Reply:
leftover from the creation process, change applied